m2e-code-quality.setup

```diff
@@ -85,6 +85,19 @@
       <sourceLocator
           rootFolder="${git.clone.location}"
           locateNestedProjects="true"/>
+      <repositoryList
+          name="Photon">
+        <repository
+            url="http://download.eclipse.org/releases/photon"/>
+        <repository
+            url="https://spotbugs.github.io/eclipse/"/>
+        <repository
+            url="http://findbugs.cs.umd.edu/eclipse/"/>
+        <repository
+            url="https://dl.bintray.com/eclipse-cs/eclipse-cs/8.0.0/"/>
+        <repository
+            url="https://dl.bintray.com/pmd/pmd-eclipse-plugin/updates/"/>
+      </repositoryList>
```

Comment:
m2e dependency is missing
<repository
            url="http://download.eclipse.org/technology/m2e/milestones/1.9"/>

Reply:
@victornoel will you add it?

Reply:
@vorburger @erwint yes, good point :)